### PR TITLE
Disable user attributes for active service

### PR DESCRIPTION
**Pull request by bodnara**
* user related attributes are disabled when adding new required attributes to active service when consents are also required

### CODE REVIEW

**Review by HejdaJakub — CHANGES_REQUESTED** [review]
Due to the async call, the attr list in the add required attr dialog is a little bit inconsistent during the loading. For a few seconds, all items are loaded, but the check if some checkboxes should be disabled isn't completed yet. So I am able for a few seconds to check also checkboxes, which will be disabled soon. 
image

File: apps/admin-gui/src/assets/config/defaultConfig.json [hunk 1]
```diff
@@ -51,6 +51,7 @@
   "skip_oidc": false,
   "group_name_secondary_regex": "^[-a-zA-Z.0-9_ ]+$",
   "group_name_error_message": "Name cannot be empty and can contain only characters a-z, A-Z, numbers, spaces, dots, '_' and '-'",
+  "enforce_consent": false,
```
[the remark this answers is not in the document]
Comment (bodnara) [reply apps/admin-gui/src/assets/config/defaultConfig.json]: Unified.

**Comment by bodnara** [comment]
> Due to the async call, the attr list in the add required attr dialog is a little bit inconsistent during the loading. 

I have set the default values to a "defensive" option and that it expects consents to be required. I also switch form `concatMap` to `mergeMap` to get `consentHub` for all facilities. It should be significantly faster as now calls can be done in parallel rather than always waiting for the previous to finish.

**Review by HejdaJakub — CHANGES_REQUESTED** [review]
Good job, it looks faster! 
I just realized one more thing. There can be a problem with the master toggle checkbox in the dialog. At this moment the `someCheckboxDiasabled` attribute is set to false (without `canBeSelected` method) so with the header checkbox, I can select also disabled checkboxes.

**Review by HejdaJakub — APPROVED** [review]

**Pull request merged**